services/tracer/TracerWithTime.cpp

```diff
@@ -10,12 +11,10 @@ namespace services
 
     void TracerWithTime::InsertHeader()
     {
-        infra::TimePoint nowTimePoint = infra::Now();
-        time_t now = std::chrono::system_clock::to_time_t(nowTimePoint);
-        std::tm* utcTime = gmtime(&now);
```

Comment:
Ouch I thought I had gotten rid of gmtime and friends a long time ago. Thanks!

Reply:
There is still one instance left... I'll see if I can remove that as well.